gcloud/connection.py

```diff
@@ -91,14 +95,23 @@ def credentials(self):
     def http(self):
         """A getter for the HTTP transport used in talking to the API.
 
-        :rtype: :class:`httplib2.Http`
-        :returns: A Http object used to transport data.
+        This will return a thread-local :class:`httplib2.Http` instance unless
+        a custom transport has been provided to the `Connection` constructor.
```

Comment:
Ditto here on two backticks (if you want to refer to `Connection` you need to use `:class:`)